.github/workflows/autobuild.yml

```diff
@@ -1,6 +1,5 @@
 #### Automatically build and upload releases to GitHub ####
 
-
 # see analyse_git_reference.py for implementation of the logic:
```

Comment:
Maybe a brief comment like

"This file builds the source and produces artifacts for every operating system. If requested it creates a release and uploads the binaries to GitHub releases. Moreover, it tests the code with CodeQl"